script/generate_sql

```diff
@@ -45,3 +45,8 @@ set -x
 
 # Record dependencies in yaml files
 ./script/bqetl dependency record --skip-existing "${SQL_DIR}"
+
+# Create the derived view schemas for the specified project
+./script/bqetl generate derived_view_schemas \
+    --target-project "${TARGET_PROJECT}" \
+    --sql-dir "${SQL_DIR}" \
```

Comment:
We'll probably want to run this in the CI as well: https://github.com/mozilla/bigquery-etl/blob/d0b831492184ad4814f86da1c27bf0d1db05548b/.circleci/config.yml#L340

Right now we are in the process of migrating away from using `scripts` to using the CLI instead (though there are a few places where the `generate_sql` script is still used)

Reply:
Discussed offline the `bqetl generate all` here https://github.com/mozilla/bigquery-etl/blob/d0b831492184ad4814f86da1c27bf0d1db05548b/.circleci/config.yml#L335 handles this